packages/neo4j-driver-lite/src/index.ts

```diff
@@ -337,7 +337,7 @@ const logging = {
     return {
       level: level,
       logger: (level: LogLevel, message: string) =>
-        console.log(`${global.Date.now()} ${level.toUpperCase()} ${message}`)
+        console.log(`${globalThis.Date.now()} ${level.toUpperCase()} ${message}`)
```

Comment:
`global` isn't defined in Deno and will cause an error.

[`globalThis`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/globalThis) works in Node (v12+) and Deno.

Reply:
Sure thing. I've pushed a commit to move it to `logging.ts` as you suggested.